### DISCUSSION TITLE
is it possible to use sid emulation when booting to a jiffydos (or other kernal)

### DISCUSSION THREAD

**Discussion by anakuron** [frntc/Sidekick64#66] [Q&A] [answered]
it seems like I can't use the sid emulation after booting to a jiffydos kernal, is there a way to make this work? and should it even be working?

when I turn the sid emulation on and then boot to jiffydos via the sidekick browser, the sound doesn't work. After booting to jiffydos I'm loading a prg through sd2iec (is there a way to load files from sidekick SD from the basic?).

if I do the same thing with the standard basic the same process works out and I have a working sound (press F3 twice from the menu, load sd2iec file browser and start the prg)

when loading prg files directly through sidekick browser the sid emulation works.

I'm using sidekick64 v0.51c with network fork v0.51 alpha5

**Comment by frntc** [comment, marked as answer]
The SID emulation is using reSID, which it turn it quite resource-consuming (large lookup tables, computation) -- this results in caches on the RPi completely spoiled. Communication of the bus, e.g. accesses to kernal ROMs, however, require reliable response times. That is, in principle I could implement this, but there's a good chance that it does not run 100% stable. (one reason why I made the SIDKick -- having sound emulation and all other Sidekick-features in parallel ;-))

**Reply by anakuron** [reply]
Thanks for explaining, as it is, it's not a huge problem I can always transfer the prg's from sd2iec to sidekick and then I get the sound working. I think it would be a nice feature to have, less swapping cables or gear, but it's not worth it if it's not stable. 

I also noticed the sound is not working for CRT's or CSDb releases that are uploaded as a zip file (probably for the same reason) the zip file is a bit odd though...

I'm definitely going to try sidkick at some point, at the moment my c64 does not have any sid at all so I've been wondering do I actually need it for this machine if I could just use sidekick's reSID for everything.

**Reply by frntc** [reply]
CRT and SID-emulation is the same conflict as kernals (essentially it's SID with anything else :-)). SIDKick is also based on reSID.
